docs/beyond_basics/101-173-contributing.rst

```diff
@@ -0,0 +1,36 @@
+^^^^^^^^^^^^^^^^
+
+* Take a look at the section :ref:`contribute` for more information.
+
+For DataLad
+^^^^^^^^^^^
+
+* **Use it!** Although it may sound nothing like a contribution, *using* DataLad is a fundamental contribution anyone can make. You can find further tutorials, materials, videos, and other resources in this handbook, and in a dedicated `Tutorials repository <https://github.com/datalad/tutorials>`_. And if you like it, you can also tell your friends, system administrators, and colleagues about it, or convince your local IT department to install it on shared compute infrastructure.
+
+* **Get in touch**: We strive to improve the clarity of DataLad and its documentation. If you tried to implement DataLad in a specific way and the existing documentation didn't make sense, or wasn't clear enough or even confusing, please help us fix it. Let us know that the instructions could have been clearer, or that it didn't cover your use case, or led you along the wrong path. And if you have suggestions for improvements, let's incorporate them! Come chat with us about what you do on `Matrix <https://app.element.io/#/room/%23datalad:matrix.org>`_ (a free, decentralized, and secure communication network), tag ``datalad`` in an issue on `Neurostars <https://neurostars.org/>`_, or get in touch via :term:`GitHub`.
+
+* **Show your support**: If you like DataLad you can show your support in various meaningful ways. You can `"star" <https://github.com/datalad/datalad/stargazers>`_ the project on GitHub. You can subscribe, like, or follow DataLad on social media: There is a `Twitter Account <https://twitter.com/datalad>`_ on which we regularly post updates, and a `YouTube channel <https://youtube.com/datalad>`_ on which we post tutorials and talks. And if you write academic papers or blog posts, you can cite the `paper about DataLad <https://joss.theoj.org/papers/10.21105/joss.03262>`_ if DataLad assisted in your work.
+
+* **Contributing on GitHub**: A most valuable contribution is your time. We are interested and grateful for opinions, bug reports, feature requests, patches, larger code contributions, or simply a notice what you use DataLad for. Find the relevant repository, be that `github.com/datalad/datalad <https://github.com/datalad/datalad>`_ (the main repository), `github.com/datalad-datasets <https://github.com/datalad-datasets>`_ (many open datasets), or any :term:`DataLad extension`, and open issues or pull requests. DataLad's `CONTRIBUTING <https://github.com/datalad/datalad/blob/master/CONTRIBUTING.md>`_ file has tons of technical and social information to get you started with code contributions. But don't be intimidated by the wealth of information you will find in there. We'll be happy to help you at any stage. Also, you can take a look at technical docs (`docs.datalad.org <http://docs.datalad.org/>`_) and in particular the :ref:`Design documents <design>` that shed light on the internal design principles of the software.
```

Comment:
I would change "Contributing on GitHub" -> "Contribute on GitHub" to stay consistent with the other section titles.

Reply:
thanks!